backend/src/apiserver/visualization/license-download.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# A script to download licenses based on
+# a specified license table (3 columns: name,license_link,license_type) csv file.
+#
+# Usage:
+#   ./license-download.sh ./third_party_licenses.csv ./third_party_licenses
+
+mkdir -p $2
+
+IFS=$'\n'
+while IFS=, read -r col1 col2 col3
+do
+  DEST="$2/$col1.LICENSE"
+  if [[ -f "$DEST" ]]; then
+    echo "Skip downloaded license file $DEST."
+  else
+    wget -O $2/$col1.LICENSE $col2
```

Comment:
add a "sleep 5" something?

Reply:
or preload and submit to our repo (since it's just LICENSE file, I think it should be OK

Reply:
Sorry, the PR was in a WIP stage, because downloads are timing out in my local environment. Just pushing these changes first so I can use a different computer to download data.

The script here doesn't run when building the container. You can see the actual error message is license missing, not timing out to github.